tests/unit/models/addon-test.js

```diff
@@ -330,12 +330,21 @@ describe('models/addon.js', function() {
 
       it('returns true when the addon name is prefixed in package.json and not in index.js', function() {
         process.env.EMBER_ADDON_ENV = 'development';
-
+        project.root = 'foo';
         project.name = () => ('@foo/my-addon');
         addon.name = 'my-addon';
         expect(addon.isDevelopingAddon(), 'addon is being developed').to.eql(true);
       });
 
+      it('displays warning if addon name is different in package.json and index.js ', function() {
+        process.env.EMBER_ADDON_ENV = 'development';
```

Comment:
This seems to never be cleaned up (you will need to add an `afterEach(function() { delete process.env.EMBER_ADDON_ENV; })` to ensure that it is cleared)...